static/src/javascripts/projects/common/modules/experiments/acquisition-test-selector.js

```diff
@@ -36,6 +37,7 @@ const isViewable = (v: Variant, t: ABTest): boolean => {
  * acquisition tests in priority order (highest to lowest)
  */
 export const acquisitionsTests: $ReadOnlyArray<AcquisitionsABTest> = [
+    supportEpicCircles, // TODO: check priority placement
```

Comment:
Can someone please confirm that it's ok to pop this in at the head of this list? Thanks.

Reply:
Can't answer this definitively but it seems fine to me.

Reply:
Yeah this isn't a particularly straightforward call to make. In this case it should be below `acquisitionsEpicUSGunCampaign` since that overlaps with this test (both apply in the US) but is more specific (it applies to pages with a specific tag) - i.e. it displays on a subset of pageviews that this test will. As things stand, `supportEpicCircles` would end up overriding the epic on the US gun campaign pages.

We could do with making this safer/more explicit/more comprehensible